pkg/routes/apitest.go

```diff
@@ -11,8 +11,8 @@ import (
 func apiCheck(w http.ResponseWriter, r *http.Request) {
 	api := stockings.IexApi{}
 	aObj, err := api.GetCurrentPrice("MELI")
-	a, _ := json.Marshal(aObj)
-	fmt.Fprintln(w, string(a))
+	//a, _ := json.Marshal(aObj)
```

Comment:
is this change intentional?

Reply:
updated so that it properly prints current price on the website